### DISCUSSION TITLE
Notifying about Emergency access in the UI and in the clients

### DISCUSSION THREAD

**Discussion by GreenBlast** [dani-garcia/vaultwarden#4351] [Ideas]
Hi. 

Thanks for this project. Really amazing.

I was testing the emergency access function and was surprised to discover that the emergency access request is almost silenced.

I would have expected some notification in the clients and also some notice or banner in the UI. In case of the emergency access account is compromised, it is very important to be notified about emergency access requests so they could be rejected on time.

I also encountered the same feature request in the Bitwarden forums.
https://community.bitwarden.com/t/client-notification-banner-etc-of-emergency-access-request/16906/1

Is there a chance we could add some banner in the UI, and add some push notification about emergency access request? I think it will be a good improvement for the current state.

**Comment by BlackDex** [comment]
E would have to wait for Bitwarden to add this to there clients.
Vaultwarden doesn't maintain those. And we can't send random message to clients.

**Reply by BlackDex** [reply]
Also, if you have mail enabled, you do receive an email about this.

**Reply by GreenBlast** [reply]
Clients for sure. But the web UI? Maybe we can add some indication. Right now it is pretty much hidden and you have to go into a specific part in the settings to be able to even check if the contact requested access.

**Reply by BlackDex** [reply]
The web-vault is also a client. Also not maintained by this project
